conf/plants/species_tree.topology.nw

```diff
@@ -177,6 +182,10 @@
             phaseolus_vulgaris,
             glycine_max,
            )Phaseoleae,
+           (
+            arachis_hypogaea_A,
+            arachis_hypogaea_B,
```

Comment:
Currently all polyploids are also in the species tree "on their own". Can you add `arachis_hypogaea` too ?